### PR TITLE
Configure CircleCI

### PR DESCRIPTION
**Pull request by VictorVelarde**
Related to #53

### Tasks
Added CircleCI v2 configuration with 3 parallel jobs:
- [x] circleci: **lint**
- [x] circleci: **test-unit:components**
- [x] circleci: **test-unit:styles**

### Current status
Tasks are launched correctly, and all **3 pass** (including test-unit:styles). It will fail in the future, when new reference images will be created (follow manual steps to fix the problem: snapshots generated at Mac local dev and remote-linux CircleCI machine may differ)

---
For some updated notes on CircleCI and reference images creation, see **DEVELOPERS.md**

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Currently with just one `reference.png`, manually copied from CircleCI

**Comment by IagoLast** [comment]
Nice job!!!  Could you add a DEVELOPERS.md with a testing section containing the issue comments?

File: packages/styles/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "lint": "stylelint 'src/**/*.scss'",
     "serve": "serve -p 5000",
     "test:watch": "echo 'No tests available yet.'",
-    "test": "node src/test.js"
+    "test": "npm run build && node src/test.js"
```
Comment (IagoLast) [inline packages/styles/package.json]: Discussion here:

we can force the build here or create a separated step in the circleCI config.

if we create a separated step the test output wont containg build traces (red mark):


screen shot 2018-07-19 at 12 39 21

Comment (VictorVelarde) [reply packages/styles/package.json]: That seems reasonable, I'll create a different step in the job

File: DEVELOPERS.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Developer Notes
```
Comment (jesusbotella) [inline DEVELOPERS.md]: ✨

**Pull request merged**